datasource/keyvaultsecret/data_acc_test.go

```diff
@@ -0,0 +1,220 @@
+// // Copyright (c) HashiCorp, Inc.
+// // SPDX-License-Identifier: MPL-2.0
+
+package keyvaultsecret
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"regexp"
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets"
```

Comment:
Do we still need to use the azure-sdk-for-go here? I assume you just haven't updated this acceptance test since migrating to the new strategy for solving this